nodeApps/generateSignedFile.js

```diff
@@ -9,46 +9,15 @@ const git = require('simple-git')()
 const releaseBranch = process.env.releaseBranch || 'staging'
```

Comment:
Not an issue but should this be in camelCase here? The other env vars (the npm ones are in snake case), and also usually they most of the time are all caps snake case, so this seems a bit mixed up convention wise.

Reply:
happy to change that